src/olympia/stats/utils.py

```diff
@@ -192,30 +194,38 @@ def get_averages_by_addon_from_bigquery(today, exclude=None):
     `{get_amo_stats_dau_view_name()}`
   WHERE
     submission_date >= @one_week_date
+  AND
+    EXTRACT(DAYOFWEEK FROM submission_date) <> 1
+  AND
+    EXTRACT(DAYOFWEEK FROM submission_date) <> 7
   GROUP BY
     addon_id),
-  three_weeks_before_this_week AS (
+  last_week AS (
```

Comment:
I understand what you mean in context, but `this_week` _is_ the last week, so having a `last_week` also is ambiguous..

Reply:
Yeah I don't love it. Any suggestions ? Maybe `this_week` and `previous_week` ?

Reply:
> Yeah I don't love it. Any suggestions ? Maybe `this_week` and `previous_week` ?

wfm